tests/integration_tests/test_connection_configuration_integration.py

```diff
@@ -1221,8 +1221,8 @@ def test_saas_connection_incorrect_secrets(
         )
         assert body["test_status"] == "failed"
         assert (
-            f"Operational Error connecting to '{mailchimp_connection_config.key}'."
-            == body["failure_reason"]
+            f"Operational Error connecting to '{mailchimp_connection_config.key}'"
+            in body["failure_reason"]
```

Comment:
Changed from a `==` to `in` since dev mode will cause the error to contain additional context.